packages/manager/apps/vrack-services/src/components/DissociateVrackModal.tsx

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  OsdsText,
+  OsdsSpinner,
+  OsdsModal,
+  OsdsButton,
+  OsdsMessage,
+} from '@ovhcloud/ods-components/react';
+import {
+  ODS_BUTTON_TYPE,
+  ODS_BUTTON_VARIANT,
+  ODS_MESSAGE_TYPE,
+  ODS_SPINNER_SIZE,
+  ODS_TEXT_LEVEL,
+  ODS_TEXT_SIZE,
+} from '@ovhcloud/ods-components';
+import { useOvhTracking } from '@ovh-ux/manager-react-shell-client';
+import { ApiError, ApiResponse } from '@ovh-ux/manager-core-api';
+import { useMutation } from '@tanstack/react-query';
+import { handleClick } from '@/utils/ods-utils';
+import { Task } from '@/api';
+import {
+  dissociateVrackServices,
+  dissociateVrackServicesQueryKey,
+} from '@/api/vrack/delete';
+
+export type DissociateVrackModalProps = {
+  headline: string;
+  description?: string;
+  isModalOpen?: boolean;
+  closeModal: () => void;
+  isLoading?: boolean;
+  dataTrackingPath?: string;
+  dataTrackingConfirmValue?: string;
+  dataTrackingCancelValue?: string;
+  vrackId: string;
+  vrackServicesId: string;
+};
+
+export const DissociateVrackModal: React.FC<DissociateVrackModalProps> = ({
+  headline,
+  description,
```

Comment:
We could put the headline and description directly in the modal, don't we ?

Those labels won't change even if we put this modal in a different page